src/keymap.cpp

```diff
@@ -498,6 +506,7 @@ static const std::map<std::string, std::uint32_t> contexts = {
 	{"podboat", KM_PODBOAT},
 	{"dialogs", KM_DIALOGS},
 	{"dirbrowser", KM_DIRBROWSER},
+	{"searchresultslist", KM_SEARCHRESULTSLIST},
```

Comment:
This new context should be documented. I believe you can just git-grep for existing contexts, and that would lead you to all the places that might need updates.

Reply:
@Minoru Do you mean add description in `doc/`? And if yes, why I need `git grep` for it?

Reply:
Yeah, I mean `doc/`. `git grep` is the simplest way to find all the places where existing contexts are mentioned, and those are the places that probably need updating now.